platform/linux-generic/pktio/netmap.c

```diff
@@ -388,13 +389,22 @@ static int netmap_open(odp_pktio_t id ODP_UNUSED, pktio_entry_t *pktio_entry,
 
 	if (pkt_nm->is_virtual) {
 		static unsigned mac;
+		uint32_t tid = syscall(SYS_gettid);
```

Comment:
Why don't you use `gettid()` function? Then you can check for its existence in `configure.ac` and provide replacement implementation.

Reply:
What's the benefit from using gettid()? It seems like the only difference is that gettid() cannot fail.

Reply:
it was copied from netmap source  I think. gittid() will generate warning due to missing glibc wrapper: https://stackoverflow.com/questions/30680550/c-gettid-was-not-declared-in-this-scope Maybe something already changed...

Reply:
@matiaselo @muvarov Hmm. I thought that there is already a Glibc wrapper. I would prefer this as a separate function, but it is of minor priority then.

Reply:
See [here](https://stackoverflow.com/questions/21279649/getting-error-in-c-program-undefined-reference-to-gettid) for an interesting discussion of why gettid() is not used.

Reply:
@lumag  we also use SYS_gettid() in shm and timer. I think that is not subject for this PR. Just general clean up.